pkiinit/pkiInit.go

```diff
@@ -63,6 +82,15 @@ func newExit(exit exit) exit {
 	return &exitCode{exit: exit}
```

Comment:
see above comment about the exitCode struct... this func would be simpler by removing the parameter